resources/views/components/delegates/favorite-toggle.blade.php

```diff
@@ -0,0 +1,31 @@
+@props(['model'])
+
+<div {{ $attributes }}>
+    <button
+        x-data="{
+            isFavorite: {{ $model->isFavorite() ? 'true' : 'false' }},
+        }"
+        type="button"
+        class="flex items-center space-x-2 font-semibold favorite-icon"
```

Comment:
add a transition so it looks better on hover

Reply:
managed to get it working with fill and stroke css. svg transitions don't seem to like vars